lms/static/scripts/frontend_apps/components/ErrorDialogApp.js

```diff
@@ -14,12 +12,12 @@ import ErrorModal from './ErrorModal';
  * (e.g. OAuth2RedirectErrorApp, LaunchErrorDialog).
  */
 export default function ErrorDialogApp() {
-  const { errorDialog } = useContext(Config);
+  const { errorDialog } = useConfig(['errorDialog']);
```

Comment:
The `errorDialog` field should always be set by the backend when this app mode is used. See `enable_error_dialog_mode`.